src/components/Popup/Popup.js

```diff
@@ -46,64 +50,92 @@ const propTypes = {
   className: PropTypes.string,
 
   /**
-   * Function triggered on close button click.
-   */
-  onCloseClick: PropTypes.func,
-
-  /**
-   * Function triggered on key up.
+   * Title HTML attributes.
    */
-  onKeyUp: PropTypes.func,
+  titles: PropTypes.shape({
+    closeButton: PropTypes.string,
+  }),
 
   /**
-   * Popup title.
+   * Function triggered on close button click.
    */
-  title: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+  onCloseClick: PropTypes.func,
 
   /**
-   * Popup padding.
+   * Hide or show the header.
    */
-  padding: PropTypes.string,
+  showHeader: PropTypes.bool,
```

Comment:
How about replacing this prop by allowing the user to set `renderHeader: () => null`? Same would apply to `showCloseButton`.

Reply:
Yes i've thought the same .

Reply:
Done